app/assets/stylesheets/crud.scss

```diff
@@ -56,3 +57,5 @@ footer {
 .icon-pencil { background-image: url('assets/pencil-square.svg'); }
 .icon-list { background-image: url('assets/list.png'); }
 .icon-zoom-in { background-image: url('assets/search.svg'); }
+.icon-export { background-image: url('assets/export.svg'); }
+.icon-close { background-image: url('assets/x.svg'); }
```

Comment:
Can we rename this file to `cancel.svg`  ? :grin: this way it atleast speaks to what it does